tasks/collect-mrrc-params/collect-mrrc-params.yaml

```diff
@@ -0,0 +1,47 @@
+---
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  name: collect-mrrc-params
+spec:
+  params:
+    - name: dataJsonPath
+      type: string
+      description: path to data json file
+  results:
+    - name: mrrcParamFilePath
+      description: path of the env file for following tasks
+    - name: cosignPubKeyConfig
+      description: the configmap name for cosign verify public key
+    - name: charonAWSSecret
+      description: the secret name for charon aws credential file
+  steps:
+    - name: collect-mrrc-params
+      image: quay.io/konflux-ci/release-service-utils:e633d51cd41d73e4b3310face21bb980af7a662f
+      script: |
+        #!/usr/bin/env bash
+        set -eux
+
+        MRRC_ENV_FILE_PATH="$(dirname $(params.dataJsonPath))/mrrc.env"
+
+        environment="$(jq -re '.mrrc.environment' $(params.dataJsonPath))"
```

Comment:
`$(params.dataJsonPath))` is the relative path to the file within the workspace. So you will need `"$(workspaces.data.path)/$(params.dataPath)"`. Maybe it's worth setting up a var for that: `DATA_FILE="$(workspaces.data.path)/$(params.dataPath)"`

Reply:
Ok. Changed.